script/universal/MultisigBase.sol

```diff
@@ -28,7 +28,7 @@ abstract contract MultisigBase is Simulator {
         uint256 nonce = safe.nonce();
         console.log("Safe current nonce:", nonce);
 
-        if (bytes(vm.envOr("SAFE_NONCE", "")).length > 0) {
+        if (bytes(vm.envOr(string("SAFE_NONCE"), string(""))).length > 0) {
```

Comment:
Not sure if this is the right fix, ran into this error earlier
```
Error (6675): Member "envOr" not unique after argument-dependent lookup in contract Vm.
  --> script/universal/MultisigBase.sol:31:19:
   |
31 |         if (bytes(vm.envOr("SAFE_NONCE", "")).length > 0) {
   |                   ^^^^^^^^

make: *** [test] Error 1
```

Reply:
Ah thanks this makes sense! I _think_ this would also work, but happy to just keep as is
```suggestion_
        if (vm.envOr(string("SAFE_NONCE"), 0)) > 0) {
```

Reply:
This gives the same error actually, pretty strange though. And no matter what I cast the 0 it doesn't work either. 
```
Error (6675): Member "envOr" not unique after argument-dependent lookup in contract Vm.
  --> script/universal/MultisigBase.sol:31:19:
   |
31 |         if (bytes(vm.envOr(string("SAFE_NONCE"), 0)).length > 0) {
```

So will keep it as is.